.github/actions/collect-info/action.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Collect information and store information'
+description: 'Collect information using EVE script and store information specified in input'
```

Comment:
What about: "Collect debug info using EVE script executed via ssh and store downloaded tarball under the specified file name"